src/SFA.DAS.AANHub.Api/Controllers/MembersController.cs

```diff
@@ -94,4 +98,35 @@ public async Task<IActionResult> GetMember(Guid memberId)
 
         return GetResponse(response);
     }
+
+    [HttpPost]
+    [Route("{memberId}/remove")]
```

Comment:
There is a requirement where a user can be restored within 14 days, which in backend terms would require updating the status to live. So I was thinking if we just change the endpoint to `members/{memberId}/status` then this endpoint will work for that scenario as well.

Reply:
done, and story changed to reflect that